app.rb

```diff
@@ -0,0 +1,27 @@
+require 'sinatra'
+require 'jwt'
+
+METABASE_EMBEDDING_SECRET = ENV.fetch('METABASE_EMBEDDING_SECRET')
+METABASE_SITE_URL = ENV.fetch('METABASE_SITE_URL') || 'http://localhost:3000'
+METABASE_EMBED_DASHBOARD_ID= ENV.fetch('METABASE_EMBED_DASHBOARD_ID')
```

Comment:
you're not aren't using this in the code.  What is it for?

Reply:
That was an oversight. I should be using that for the resource ID in the JWT payload.